tags.go

```diff
@@ -586,18 +585,14 @@ func encodeTags(tags []string) ([]string, error) {
 				return nil, fmt.Errorf("invalid base64 tag key: %v %w",
 					key, err)
 			}
+		}
 
-			rk = string(key)
-
-			if keyTest.Match(key) {
-				rk = `b"` + base64.StdEncoding.EncodeToString(key) +
-					`"`
-			}
-		} else {
-			rk = key
+		if !keyTest.MatchString(key) {
```

Comment:
we support base64 encoded tag namespaces?

Reply:
Yep. Those are called categories in our tag nomenclature. And yes, we definitely do.

Reply:
Matter of fact, the variable naming in that function is bad, because categories are definitely not keys. Just a force of habit. I'm updating the PR to change this.

Reply:
cool, thank you! Definitely one of the things that threw me off when I first started, and will certainly make an improvement in readability.